yamux/Cargo.toml

```diff
@@ -20,3 +20,4 @@ pin-project = "1.1.0"
 
 [dev-dependencies]
 quickcheck = "1.0"
+futures = { version = "0.3.12", default-features = false, features = ["executor"] }
```

Comment:
`executor` is needed for tests. Is there a better way to enable the feature for testing only?

Reply:
Nope, adding the dependency with the features required for `dev-dependencies` is the best I know of.